_data/master/navbars/getting-started.yml

```diff
@@ -143,16 +143,26 @@ toc:
       path: /getting-started/openstack/installation/ubuntu
     - title: Red Hat Enterprise Linux 
       path: /getting-started/openstack/installation/redhat
-    - title: Juju Charms
-      path: /getting-started/openstack/installation/juju
-    - title: Fuel
-      path: /getting-started/openstack/installation/fuel
     - title: DevStack
       path: /getting-started/openstack/installation/devstack
-    - title: Chef
-      path: /getting-started/openstack/installation/chef
   - title: Upgrade
```

Comment:
I think it would be good to keep symmetry with Kubernetes, in terms of the sequence, where "Upgrade" is at the end of the list.